diff --git a/src/atheme/PKGBUILD b/src/atheme/PKGBUILD
index d4a9580..33175ec 100644
--- a/src/atheme/PKGBUILD
+++ b/src/atheme/PKGBUILD
@@ -7,7 +7,7 @@ _pcre=default
 _perl=default
 _contrib=default
 _cracklib=default
-_ldap=no
+_ldap=default
 
 pkgname=atheme
 pkgver=7.2.10_r2
diff --git a/src/biboumi-git/PKGBUILD b/src/biboumi-git/PKGBUILD
index 626ffec..08c7b70 100644
--- a/src/biboumi-git/PKGBUILD
+++ b/src/biboumi-git/PKGBUILD
@@ -2,7 +2,7 @@
 
 _pkgname=biboumi
 pkgname="$_pkgname-git"
-pkgver=8.2.r83.g3621458
+pkgver=8.2.r74.gcd20177
 pkgrel=1
 pkgdesc="XMPP gateway to IRC"
 arch=('i686' 'x86_64' 'armv7h' 'aarch64')
diff --git a/src/broadcom-bt-firmware-git/PKGBUILD b/src/broadcom-bt-firmware-git/PKGBUILD
index 75eefa4..3c570a3 100644
--- a/src/broadcom-bt-firmware-git/PKGBUILD
+++ b/src/broadcom-bt-firmware-git/PKGBUILD
@@ -4,8 +4,8 @@ pkgname=broadcom-bt-firmware-git
 pkgdesc="Broadcom bluetooth firmware."
 conflicts=('bcm4335c0-firmware' 'bcm4350c5-firmware' 'bcm4356a2-firmware' 'bcm20702a1-firmware' 'bcm20702b0-firmware' 'bcm20703a1-firmware' 'bcm43142a0-firmware')
 provides=('bcm4335c0-firmware' 'bcm4350c5-firmware' 'bcm4356a2-firmware' 'bcm20702a1-firmware' 'bcm20702b0-firmware' 'bcm20703a1-firmware' 'bcm43142a0-firmware')
-pkgver=12.0.1.1011_p1.r4.g632c15f
-pkgrel=1
+pkgver=12.0.1.1011_p1.r3.gc0bd928
+pkgrel=2
 arch=('any')
 makedepends=('git')
 url="https://github.com/winterheart/broadcom-bt-firmware"
diff --git a/src/chocolate-doom-git/PKGBUILD b/src/chocolate-doom-git/PKGBUILD
index 319c0ae..f96dbdf 100644
--- a/src/chocolate-doom-git/PKGBUILD
+++ b/src/chocolate-doom-git/PKGBUILD
@@ -3,7 +3,7 @@
 pkgname=(chocolate-{doom,heretic,hexen,strife,common}-git)
 _pkgname=${pkgname[0]/-git/}
 pkgdesc="Historically-accurate Doom, Heretic, Hexen, and Strife ports."
-pkgver=3.0.0.r804.485b939b
+pkgver=3.0.0.r58.a8985916
 pkgrel=1
 arch=('i686' 'x86_64')
 url="http://www.chocolate-doom.org/"
diff --git a/src/dino-git/PKGBUILD b/src/dino-git/PKGBUILD
index f85dd70..5ac9b14 100644
--- a/src/dino-git/PKGBUILD
+++ b/src/dino-git/PKGBUILD
@@ -2,7 +2,7 @@
 # Contributor: svalo <me@valo.space>
 
 pkgname=dino-git
-pkgver=r487.1853ee8
+pkgver=r370.38b5320
 pkgrel=1
 pkgdesc="Simple and modern Jabber/XMPP client written in vala"
 arch=('i686' 'x86_64' 'aarch64')
@@ -33,7 +33,6 @@ pkgver() {
 build() {
     cd "${srcdir}/${pkgname%-git}"
     ./configure $_DISABLE_PLUGINS $_ENABLE_PLUGINS --prefix="/usr" --with-tests CC="$CC" CFLAGS="$CFLAGS" VALACFLAGS="$VALACFLAGS"
-    sed -i '/SrvRecordsTls/d' libdino/src/service/module_manager.vala
     make
 }
 
diff --git a/src/emulationstation-git/PKGBUILD b/src/emulationstation-git/PKGBUILD
index cb67560..19338ed 100644
--- a/src/emulationstation-git/PKGBUILD
+++ b/src/emulationstation-git/PKGBUILD
@@ -6,7 +6,7 @@ pkgname=emulationstation-git
 _gitname=EmulationStation
 pkgrel=1
 epoch=1
-pkgver=2.4.1.r429.g13819ec0
+pkgver=2.4.1.r392.gb7bec0eb
 pkgdesc="A graphical front-end for emulators with controller navigation. Developed for the Raspbery Pi, but runs on most Linux systems."
 arch=('i686' 'x86_64' 'armv6h' 'armv7h')
 url="https://github.com/RetroPie/EmulationStation"
diff --git a/src/hg-git-hg/PKGBUILD b/src/hg-git-hg/PKGBUILD
index 5d3a9c0..48b1bd2 100644
--- a/src/hg-git-hg/PKGBUILD
+++ b/src/hg-git-hg/PKGBUILD
@@ -7,7 +7,7 @@
 #
 name=hg-git
 pkgname=$name-hg
-pkgver=1169.2135ddef6d6e
+pkgver=631.711baa274f78
 pkgrel=1
 pkgdesc="the Hg-Git mercurial plugin"
 arch=(any)
diff --git a/src/kiwiirc/PKGBUILD b/src/kiwiirc/PKGBUILD
index 0beb339..e679c19 100644
--- a/src/kiwiirc/PKGBUILD
+++ b/src/kiwiirc/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Jingbei Li <i@jingbei.li>
 # Contributor: Jack Allnutt <jack@allnutt.eu>
 pkgname=kiwiirc
-pkgver=1.1.0.r2724.g458c5bbd
+pkgver=1.1.0.r2044.g4c979963
 pkgrel=1
 pkgdesc="A hand-crafted web-based IRC client that you can enjoy"
 arch=('any')
diff --git a/src/movim-git/PKGBUILD b/src/movim-git/PKGBUILD
index da0e13a..108304d 100644
--- a/src/movim-git/PKGBUILD
+++ b/src/movim-git/PKGBUILD
@@ -3,7 +3,7 @@
 
 _pkgbase='movim'
 pkgname=movim-git
-pkgver=r7308.8a32ba694
+pkgver=r7476.b0934832f
 pkgrel=1
 pkgdesc="Movim is a decentralized social network, written in PHP and HTML5 and based on the XMPP standard protocol."
 arch=('any')
@@ -53,10 +53,10 @@ package() {
 
   install -m755 -d "$pkgdir/usr/share/webapps/$_pkgbase"
 
-  cp -r app database lib locales src theme vendor \
+  cp -r app database lib locales public src vendor \
     "$pkgdir/usr/share/webapps/$_pkgbase"
-  install -Dm644 VERSION CHANGELOG.md INSTALL.md README.md index.php \
-    linker.php phinx.php "$pkgdir/usr/share/webapps/$_pkgbase"
+  install -Dm644 CHANGELOG.md doap.xml INSTALL.md linker.php phinx.php \
+    README.md VERSION "$pkgdir/usr/share/webapps/$_pkgbase"
   install -Dm755 daemon.php "$pkgdir/usr/share/webapps/$_pkgbase"
 
   # Configuration file
@@ -68,4 +68,4 @@ sha256sums=('SKIP'
             '5dfff91dd4a54f3d3713530e204370a96d37898b670a61123d8cad42f92da306'
             '813d580300ee5bb7f1f9603db2404f5af67fd02f49017708d4adb919d4422045'
             'c8b569f1eafb97d8d03bbaabb9b4d6ed3415f922a05a52bd865942676368f4c5'
-            'e3cd2ceb71f4ef4689028313188def52aa19859e35aa8abcaa200495cd187935')
+            'd7847c2b5d9897122168c66493bc72200432c2b15317e38afbfb06a8500257a2')
diff --git a/src/openconnect-git/PKGBUILD b/src/openconnect-git/PKGBUILD
index 55ccfbd..bedbece 100644
--- a/src/openconnect-git/PKGBUILD
+++ b/src/openconnect-git/PKGBUILD
@@ -2,7 +2,7 @@
 
 pkgname=openconnect-git
 _pkgname=openconnect
-pkgver=8.03.r53.g2b9bc4d9
+pkgver=7.08.r127.g8977877
 pkgrel=1
 pkgdesc="Open client for Cisco AnyConnect VPN"
 arch=('i686' 'x86_64')
diff --git a/src/prosody-hg-stable/PKGBUILD b/src/prosody-hg-stable/PKGBUILD
index 6517730..6df4c49 100644
--- a/src/prosody-hg-stable/PKGBUILD
+++ b/src/prosody-hg-stable/PKGBUILD
@@ -8,7 +8,7 @@
 
 pkgname=prosody-hg-stable
 pkgrel=1
-pkgver=0.10.r7593+.7e053c022782+
+pkgver=0.10.r7499+.3d21c63ec03f+
 pkgver() {
   cd "$srcdir/prosody-hg"
   printf "0.10.r%s.%s" "$(hg identify -n)" "$(hg identify -i)"
diff --git a/src/python2-axolotl-curve25519-git/PKGBUILD b/src/python2-axolotl-curve25519-git/PKGBUILD
index 5423d86..cf6af48 100644
--- a/src/python2-axolotl-curve25519-git/PKGBUILD
+++ b/src/python2-axolotl-curve25519-git/PKGBUILD
@@ -3,7 +3,7 @@
 
 pkgname='python2-axolotl-curve25519-git'
 _pkgname='python-axolotl-curve25519'
-pkgver=0.4.1.2.r0.g293f9cd
+pkgver=r14.e4a9c4d
 pkgrel=1
 pkgdesc="Python wrapper for curve25519 library"
 url="https://github.com/tgalal/${_pkgname}"
diff --git a/src/python2-axolotl-git/PKGBUILD b/src/python2-axolotl-git/PKGBUILD
index dccbe50..948fff2 100644
--- a/src/python2-axolotl-git/PKGBUILD
+++ b/src/python2-axolotl-git/PKGBUILD
@@ -3,7 +3,7 @@
 
 pkgname='python2-axolotl-git'
 _pkgname='python-axolotl'
-pkgver=0.2.3.r0.gcc82fd5
+pkgver=0.1.39.r0.g37c5c55
 pkgrel=1
 pkgdesc="Python port of libaxolotl"
 url="https://github.com/tgalal/python-axolotl"
diff --git a/src/qemu-user-static/PKGBUILD b/src/qemu-user-static/PKGBUILD
index 4c500a2..0d10cea 100644
--- a/src/qemu-user-static/PKGBUILD
+++ b/src/qemu-user-static/PKGBUILD
@@ -32,7 +32,6 @@ build() (
   cd build-user-static
 
   ../qemu-${pkgver}/configure \
-    --python=/bin/python2 \
     --prefix=/usr \
     --sysconfdir=/etc \
     --localstatedir=/var \
diff --git a/src/redis-desktop-manager/PKGBUILD b/src/redis-desktop-manager/PKGBUILD
index bbcfb25..3fbc74e 100644
--- a/src/redis-desktop-manager/PKGBUILD
+++ b/src/redis-desktop-manager/PKGBUILD
@@ -18,7 +18,7 @@ depends=(
   'qt5-graphicaleffects'
   'qt5-svg'
   'libssh2')
-makedepends=('git' 'gcc' 'python2' 'python')
+makedepends=('git' 'gcc' 'python2')
 conflicts=('redis-desktop-manager-bin')
 source=("rdm::git://github.com/uglide/RedisDesktopManager.git#tag=$pkgver"
         'rdm.sh'
diff --git a/src/rootmp-hook/PKGBUILD b/src/rootmp-hook/PKGBUILD
index b2504dd..e6620b1 100755
--- a/src/rootmp-hook/PKGBUILD
+++ b/src/rootmp-hook/PKGBUILD
@@ -8,7 +8,6 @@ arch=(any)
 license=('GPL3')
 install=${pkgname}.install
 depends=('util-linux' 'rsync')
-makedepends=('git')
 source=(${pkgname%-git}::git+http://github.com/jim945/rootmp-hook.git)
 
 package() {
diff --git a/src/ternimal-git/PKGBUILD b/src/ternimal-git/PKGBUILD
index fc92d3a..ba2b5ed 100644
--- a/src/ternimal-git/PKGBUILD
+++ b/src/ternimal-git/PKGBUILD
@@ -1,8 +1,8 @@
 # Maintainer: xiretza <xiretza+aur@gmail.com>
 _pkgname=ternimal
 pkgname="${_pkgname}-git"
-pkgver=r9.e7953b4
-pkgrel=1
+pkgver=r5.2eea4f4
+pkgrel=2
 pkgdesc="Simulate a lifeform in the terminal"
 arch=('x86_64' 'i686' 'armv7h')
 url="https://github.com/p-e-w/ternimal"
diff --git a/src/zpaq/PKGBUILD b/src/zpaq/PKGBUILD
index 875e4a6..d928d48 100644
--- a/src/zpaq/PKGBUILD
+++ b/src/zpaq/PKGBUILD
@@ -32,10 +32,10 @@ build()
 	fi
 
 	msg 'Building libzpaq'
-	$CXX $CXXFLAGS $LDFLAGS -fPIC -O3 -shared -Dunix -DNDEBUG libzpaq.cpp -o libzpaq.so
+	$CXX $CXXFLAGS $LDFLAGS -fPIC -shared -Dunix -DNDEBUG libzpaq.cpp -o libzpaq.so
 
 	msg 'Building zpaq'
-	$CXX ${CXXFLAGS/-flto} ${LDFLAGS/-flto} -O3 -pthread -Dunix -DNDEBUG zpaq.cpp -L. -lzpaq -o zpaq
+	$CXX ${CXXFLAGS/-flto} ${LDFLAGS/-flto} -pthread -Dunix -DNDEBUG zpaq.cpp -L. -lzpaq -o zpaq
 
 	msg 'Building man page'
 	pod2man zpaq.pod zpaq.1
